lib/utils.sh

```diff
@@ -43,12 +66,12 @@ list_installed_versions() {
   plugin_path=$(get_plugin_path "$plugin_name")
 
   local plugin_installs_path
-  plugin_installs_path=$(asdf_dir)/installs/${plugin_name}
+  plugin_installs_path="$(asdf_data_dir)/installs/${plugin_name}"
 
   if [ -d "$plugin_installs_path" ]; then
-	# shellcheck disable=SC2045
+  # shellcheck disable=SC2045
     for install in $(ls -d "${plugin_installs_path}"/*/ 2>/dev/null); do
-		basename "$install" | sed 's/^ref-/ref:/'
+    basename "$install" | sed 's/^ref-/ref:/'
```

Comment:
Indentation seems off here.

Reply:
There are a mix of tabs and spaces :S , maybe my editor messed it up